extra/mariabackup/xtrabackup.cc

```diff
@@ -1732,11 +1739,13 @@ static int prepare_export()
       " --defaults-extra-file=./backup-my.cnf --defaults-group-suffix=%s --datadir=."
       " --innodb --innodb-fast-shutdown=0 --loose-partition"
       " --innodb_purge_rseg_truncate_frequency=1 --innodb-buffer-pool-size=%llu"
-      " --console  --skip-log-error --skip-log-bin --bootstrap  < "
+      " --console --skip-log-error --skip-log-bin --bootstrap"
+      " --innodb-force-recovery=%lu < "
       BOOTSTRAP_FILENAME IF_WIN("\"",""),
-      mariabackup_exe, 
+      mariabackup_exe,
       orig_argv1, (my_defaults_group_suffix?my_defaults_group_suffix:""),
-      xtrabackup_use_memory);
+      xtrabackup_use_memory,
+      srv_force_recovery);
```

Comment:
Do not output `--innodb-force-recovery=0` to keep compatible with older versions.

Side note: I think that @vaintroub could have made the `--defaults-file` branch unnecessary by including the English language strings directly in the executables, a while ago.

Side note: The `else` branch is using `sprintf` instead of the more appropriate `snprintf`. You could fix that as well.

Reply:
I am not familiar with why the branch exists (if you would like me to remove it, please let me know). I will change that to `snprintf` though.